spec/abstractops.html

```diff
@@ -651,40 +651,190 @@ <h1>
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-temporal-roundhalfawayfromzero" aoid="RoundHalfAwayFromZero">
-    <h1>RoundHalfAwayFromZero ( _x_ )</h1>
+  <emu-clause id="sec-temporal-getunsignedroundingmode" type="abstract operation">
+    <h1>
+      GetUnsignedRoundingMode (
```

Comment:
Question: With this AO landing in Temporal 262, I assume that Intl.NumberFormat should reference this AO instead of including it in its own proposal?

If NFv3 lands first (which seems likely -- it's late Stage 3), should it add these AOs into 262, or should it keep them in 402, and Temporal moves them?

Reply:
Either approach seems workable, but NFv3 should indeed contain an inline implementation of the AO.

Reply:
It seems less work overall if NFv2 adds these operations into 262 already.

Reply:
I think I prefer NFv3 keeping these AOs in 402, as planned, and Temporal moves them, because NFv3 does not currently touch 262 at all, but Temporal is touching both 402 and 262. It seems easy enough to add it as a diff here.